test/mock/mock.repository.ts

```diff
@@ -17,10 +17,14 @@ export class MockRepository<Entity> {
         this.manager = new MockEntityManager<Entity>(this);
     }
 
+    // Disabling this below because the conditions param is required for mocking for some reason. This is a good future issue to investigate.
+    /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Nit: You could also use `// eslint-disable-next-line @typescript-eslint/no-unused-vars`

Reply:
Oh yeah, good catch!